tests/sentry/eventstream/kafka/test_dispatch.py

```diff
@@ -116,6 +116,7 @@ def test_dispatch_task_with_occurrence(mock_post_process_group: Mock) -> None:
     assert mock_post_process_group.call_args.kwargs == {
         "kwargs": {
             "cache_key": "e:066f15fe1cd2406aaa7c6a07471d7aef:2",
+            "eventstream_type": None,
```

Comment:
why is it none?

Reply:
None is the default value https://github.com/getsentry/sentry/pull/79654/files/ab36e1bc7375285c9d7f3d824285539eb5bfacaf#diff-362eb7bd2f99e90fc453a0d53c450ff561944cf2ab533f6157ac664f9f2d16f3R48